DigitalOffice/Services/FileService/Mappers/FileMapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using FileService.Database.Entities;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using System;
+
+namespace FileService.Mappers
+{
+    public class FileMapper : IMapper<DbFile, FileRequest>, IMapper<FileRequest, DbFile>
+    {
+        public FileRequest Map(DbFile dbFile)
+        {
+            return new FileRequest()
+            {
+                Id = dbFile.Id,
+                Content = Convert.ToBase64String(dbFile.Content),
+                ContentExtension = dbFile.ContentExtension,
+                Name = dbFile.Name
+            };
+        }
+        public DbFile Map(FileRequest file)
+        {
+            return new DbFile()
+            {
+                Id = file.Id,
```

Comment:
Add new property to DbFile -> DateTime Added = DateTime.UtcNow

Reply:
Should we add this to FileRequest as well? Does front-end need to know when the file was added to Db?
Something like "you uploaded this on 01.01.01", or "uploaded on 31.12.2000"

Reply:
> Should we add this to FileRequest as well? Does front-end need to know when the file was added to Db?
> Something like "you uploaded this on 01.01.01", or "uploaded on 31.12.2000"

Do you think about the situation "when the backend lags"?

Reply:
> > Should we add this to FileRequest as well? Does front-end need to know when the file was added to Db?
> > Something like "you uploaded this on 01.01.01", or "uploaded on 31.12.2000"
> 
> Do you think about the situation "when the backend lags"?

Didnt quite get your point. What does that situation mean?
You mean that the Date.Now should come from frontend and if its not there then we program the time by ourselves?

Reply:
Not. I thought that you were thinking, what if after sending the file it does not reach the service for a long time. Should we in this case indicate the "time of sending to add" or "time of adding".

And I tried to find out what you meant.

Reply:
No need, just in dbModel at this moment and use DateTime.**UtcNow**